src/database/repositories/StatusRepository.js

```diff
@@ -0,0 +1,28 @@
+module.exports = class StatusRepository {
+  constructor(statusModal) {
+    this.statusModal = statusModal;
+  }
+
+  async CreateOrUpdate(shardID, ping, lastPingAt, guilds, uptime) {
+    const result = await this.statusModal.findById(shardID);
+    if (result) {
+      this.statusModal.updateOne({ _id: shardID }, {
+        $set: {
+          ping, lastPingAt, guilds, uptime,
+        },
+      });
+    } else {
+      this.statusModal.create({
+        _id: shardID, ping, lastPingAt, guilds, uptime,
+      });
+    }
+  }
+
+  addMaintenance(commandName) {
+    this.statusModal.updateOne({ _id: 'main' }, { $push: { disabledCommands: commandName } });
+  }
```

Comment:
```suggestion
  async addMaintenance(commandName) {
    await this.statusModal.updateOne({ _id: 'main' }, { $push: { disabledCommands: commandName } });
  }
```

Reply:
I made it without async/await first, and tested it. And it doesn't work. The queries weren't executed without the await. So i put the await, tested again and the queries where executed